recipes/django-bootstrap5/LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The LICENSE file is part of the pypi package. You can remove it from the recipe.

Reply:
I've removed the license file from the recipe.